Dockerfiles/mlpack-dependencies-debian/Dockerfile

```diff
@@ -23,32 +22,41 @@ ADD armadillo-code /armadillo-code
 WORKDIR armadillo-code
 RUN cmake . && make && make install && rm -rf /armadillo-code
 
+# Installing Boost from source
+ADD boost_1_49_0 /boost_1_49_0
```

Comment:
I think this might be part of the permissions problem.  It looks like when this is added, most of the files are not world-readable.  Can you tell me more about what the source you are adding is?